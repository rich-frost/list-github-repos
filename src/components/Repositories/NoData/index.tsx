import { Tr, Td, Icon, Center } from '@chakra-ui/react';
import { BiError } from 'react-icons/bi';

export default function NoData() {
  return (
    <Tr>
      <Td colSpan={3}>
        <Center>
          <Icon as={BiError} mx="5px" /> No repositories found
        </Center>
      </Td>
    </Tr>
  );
}
