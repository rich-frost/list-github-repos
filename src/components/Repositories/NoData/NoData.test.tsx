import renderer from 'react-test-renderer';
import NoData from './index';
import { Table, Tr } from '@chakra-ui/react';

describe('NoData component', () => {
  it('renders the component', () => {
    const component = renderer.create(
      <Table>
        <Tr>
          <NoData />
        </Tr>
      </Table>
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
