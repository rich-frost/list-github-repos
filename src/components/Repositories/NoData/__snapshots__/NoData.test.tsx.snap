// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoData component renders the component 1`] = `
<table
  className="chakra-table css-0"
>
  <tr
    className="css-0"
  >
    <tr
      className="css-0"
    >
      <td
        className="css-0"
        colSpan={3}
      >
        <div
          className="css-gmuwbf"
        >
          <svg
            className="chakra-icon css-av9skc"
            fill="currentColor"
            focusable={false}
            height="1em"
            stroke="currentColor"
            strokeWidth="0"
            style={
              {
                "color": undefined,
              }
            }
            viewBox="0 0 24 24"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M11.001 10h2v5h-2zM11 16h2v2h-2z"
            />
            <path
              d="M13.768 4.2C13.42 3.545 12.742 3.138 12 3.138s-1.42.407-1.768 1.063L2.894 18.064a1.986 1.986 0 0 0 .054 1.968A1.984 1.984 0 0 0 4.661 21h14.678c.708 0 1.349-.362 1.714-.968a1.989 1.989 0 0 0 .054-1.968L13.768 4.2zM4.661 19 12 5.137 19.344 19H4.661z"
            />
          </svg>
           No repositories found
        </div>
      </td>
    </tr>
  </tr>
</table>
`;
